File: Modul 7/soal 1/PRAK601-2210817220020-Helena Rosalina.py
kolom, baris=map(int, input().split())
bilangan= list(map(int,input().split()))
matriks=[]
a=0
for i in range(kolom) :
    matriks.append(bilangan[a:a+baris])
    a= a+baris
for kolom in matriks:
    for bilangan in kolom:
        print(bilangan,end=' ')
    print()

File: Modul 7/soal 1/PRAK601-2210817220020-Helena Rosalina.c
#include <stdio.h>

int main ()
{

int baris,kolom,a,b,isi;
scanf("%d %d", &baris,&kolom);
int bilangan[baris][kolom] ;
for (a=0;a<baris;a++){
    for (b=0;b<kolom;b++){
        scanf("%d", &isi);
        bilangan[a][b] = isi;
        }
    }
printf("\n");
for (a=0;a<baris;a++){
    for (b=0;b<kolom;b++){
        printf("%d ",bilangan [a][b]);
        }
    printf("\n");
    }
return 0;
}
